components/org.wso2.carbon.siddhi.editor.core/src/main/resources/web/commons/lib/theme-wso2-2.0.0/extensions/datatables/js/dataTables.wso2.js

```diff
@@ -0,0 +1,327 @@
+/*
+ ~   Copyright (c) WSO2 Inc. (http://wso2.com) All Rights Reserved.
+ ~
+ ~   Licensed under the Apache License, Version 2.0 (the "License");
+ ~   you may not use this file except in compliance with the License.
+ ~   You may obtain a copy of the License at
+ ~
+ ~        http://www.apache.org/licenses/LICENSE-2.0
+ ~
+ ~   Unless required by applicable law or agreed to in writing, software
+ ~   distributed under the License is distributed on an "AS IS" BASIS,
+ ~   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ ~   See the License for the specific language governing permissions and
+ ~   limitations under the License.
+*/
+
+/*
+ ~   DataTables WSO2 Integration
+ ~   ©2016 WSO2 Inc. - datatables.net/license
+ ~
+ ~   DataTables integration for WSO2 Visual Elements. This requires WSO2 Visual Elements 1.0.0 and
+ ~   DataTables 1.10 or newer.
+ */
+
+(function(window, document, undefined){
+
+    var factory = function($, DataTable) {
+        "use strict";
+        
+        /* Set the defaults for DataTables initialisation */
+        $.extend(true, DataTable.defaults, {
+            bSortCellsTop: true,
+            responsive: false,
+            autoWidth: false,
+            dom: '<"dataTablesTop"' +
+                'f' +
+                '<"dataTables_toolbar">' +
+                '>' +
+                'rt' +
+                '<"dataTablesBottom"' +
+                'lip' +
+                '>',
+            language: {
+                searchPlaceholder: 'Filter by ...',
+                search: ''
+            }
+        });
+        
+        var wso2Extend = function(settings, opts) {
+
```

Comment:
@erangatl shall we remove the extra spaces

Reply:
Fixed with #1690